### DISCUSSION TITLE
Public feed Reddit-like with Hot, New, Top and Reply

### DISCUSSION THREAD

**Discussion by danrobi11** [beakerbrowser/beaker#1574] [General]
Hello 
Anything on roadmap about a sort of 'Public Feed' ? 
Maybe reddit-like with example: Hot, New, Top 
Also about replying to blog post? 
Any thoughts?

**Comment by ghost** [comment]
Hacker News' style is also great alternative!

**Comment by pfrazee** [comment]
At the moment we've got a twitter-like via blahbity blog, which is probably the userland app I'll develop most. A link aggregator could work fairly similarly. I think somebody should be able to build it.

**Comment by pmario** [comment]
>blahbity blog

Is there an explanation to the name?

**Reply by pfrazee** [reply]
It was originally a throwaway pun and then it stuck. It's a microblogging aggregator.

We may need to rename it.

**Reply by Pawper** [reply]
I think folks just don't get the reference, but I like it. https://en.wiktionary.org/wiki/blabbity

**Reply by stehuebn** [reply]
> It was originally a throwaway pun and then it stuck. It's a microblogging aggregator.
> 
> We may need to rename it.

Actually I like that name! Just my 2 cents...
